WordPress/twentytwentytwo
# Hide "Comments Closed" message

**Description**

This PR is an alternative to #127 per some discussion in this trac ticket https://core.trac.wordpress.org/ticket/54283 (thanks @audrasjb for opening). It depends on https://github.com/WordPress/gutenberg/pull/35743

**Testing Instructions** 

1. Check out the above Gutenberg PR and build it.
2. Go to a post or page whose comments are disallowed
3. Verify the message is hidden
